src/extension.ts

```diff
@@ -156,6 +157,8 @@ export async function activate(context: vscode.ExtensionContext) {
 		context.subscriptions.push(watcher)
 	}
 
+	maybeRunAutoLaunchPrompt(context) // kilocode_change
```

Comment:
Should there be await here?

Reply:
I didn't want to block the rest of the extension loading, but I agree it's a little confusing because it is an async function. Maybe I could name it better?